db/migrate/20210220065508_add_receive_approval_notifications_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddReceiveApprovalNotificationsToUsers < ActiveRecord::Migration[5.2]
+  def change
+    add_column :users, :receive_approval_notifications, :boolean, null: false, default: true
```

Comment:
Need to exclude this field from sync APIs. I think we only send users in the teleconsult MOs endpoint.

Reply:
Ah, interesting. Will do. Should our API specs fail when unexpected fields are added to the response?

Reply:
Our "transformer" sends a whitelist of attributes, so we are all good. I've updated the transformer spec as well to perform a strict check. If something gets in there accidentally, the spec should fail.